internal/builders/nodejs/README.md

```diff
@@ -212,36 +212,26 @@ The Node.js builder accepts the following inputs:
+| node-version      | No       |                    | The version of Node.js to use. If no value is supplied, the `node` version from `$PATH` is used.                                                                                                                                                    |
+| node-version-file | No       |                    | File containing the version Spec of the version to use. Examples: .nvmrc, .node-version, .tool-versions.                                                                                                                                            |
+| rekor-log-public  | No       | false              | Set to true to opt-in to posting to the public transparency log. Will generate an error if false for private repositories. This input has no effect for public repositories. See [Private Repositories](#private-repositories).<br>Default: `false` |
+| run-scripts       | No       |                    | A comma separated ordered list of npm scripts to run before running `npm publish`. See [scripts](https://docs.npmjs.com/cli/v9/using-npm/scripts) for more information. \                                                                           |
 
 ### Workflow Outputs
 
 The Node.js builder produces the following outputs:
 
-| Name                    | Description                                                                                                    |
-| ----------------------- | -------------------------------------------------------------------------------------------------------------- |
-| artifact-name           | The name of the package artifact uploaded to the workflow run.                                                 |
-| provenance-name         | The name of the provenance attestation uploaded to the workflow run.                                           |
-| release-asset-name      | The package asset name uploaded to the release (if upload-assets is true). e.g. sigstore-1.1.1.tgz             |
-| release-id              | The name of the release where the package tarball and provenance were uploaded (if upload-assets is true).     |
-| release-provenance-name | The provenance asset name uploaded to the release (if upload-assets is true). e.g. sigstore-1.1.1.intoto.jsonl |
+| Name                       | Description                                                            |
+| -------------------------- | ---------------------------------------------------------------------- |
+| package-name               | The file name of the package tarball in the upload artifact.           |
+| package-download-name      | The name of the package artifact uploaded to the workflow run.         |
+| package-download-sha256    | The sha256 of the package artifact uploaded to the workflow run.       |
+| provenance-name            | The file name of the attestation upload artifact.                      |
```

Comment:
nit: 'provenance attestation' to be consistent with the output name

Reply:
Done.